src/components/SavedSongs/SavedSongList.js

```diff
@@ -21,26 +22,17 @@ export default function SavedSongList({
   return (
     <ListStyled>
```

Comment:
You habe no naming conflict, so the `Styled` postfix is not needed, maybe simply call it `List` or `SongsList` for expressiveness?